stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/logic/internal/LogicRegistry.kt

```diff
@@ -128,11 +128,29 @@ internal class LogicRegistry internal constructor(
         }
     }
 
+    /**
```

Comment:
Also non conflicting with the `v6` version of this feature seen in the following pr: https://github.com/GetStream/stream-chat-android/pull/4639/files#diff-bdd7ddb84df9b26d6bb6f97be0c1f459d03af78461571cb22937590ac4293eb5